docs/arcaflow/index.md

```diff
@@ -21,149 +22,138 @@ You can use Arcaflow for many things. We use it for:
+    **How are Arcaflow and Tekton similar?**
 
-    - It has tasks and pipelines to run steps in sequence
-    - You can pass data between steps
+    - They both have tasks and pipelines to run steps in sequence.
+    - They both allow for passing of data between steps.
 
-    **What it does better than Arcaflow**
-    
-    - It is a full CI/CD system that can be coupled to your version control system without additional tools
-    - Built-in supply chain security
-    - Any container image can be run without additional integrations
+    **How is Tekton different?**
     
-    **What it does worse than Arcaflow**
-    
-    - It is deployed inside a Kubernetes cluster and cannot exist without it
-    - No officially supported integrations with other platforms and tools
-    - It does not have a typing system and cannot guarantee type-safety or validation of data
+    - Tekton is a full CI/CD system that can be coupled to your version control system without additional tools.
+    - Tekton has built-in supply chain security.
+    - Tekton allows you to run any container image without additional integrations.
+    - Tekton is deployed inside a Kubernetes cluster and cannot exist without it.
+    - Tekton has no officially-supported integrations with other platforms and tools.
 
 ??? "qDup"
     
-    [qDup](https://github.com/Hyperfoil/qDup) is a workflow engine for shell commands.
-
-    **How it is similar to Arcaflow**
-
-    - It's a workflow system
-    - It runs on your laptop and doesn't need a large server
-    - You don't need to deploy it or run it permanently
-    - It can run things on remote systems out of the box
+    [qDup](https://github.com/Hyperfoil/qDup) allows shell commands to be *queued up* across multiple servers to coordinate performance tests. It is designed to follow the same workflow as a user at a terminal so that commands can be performed with or without qDup. Commands are grouped into re-usable scripts that are mapped to different hosts by roles.
 
-    **What it does better than Arcaflow**
+    **How are Arcaflow and qDup similar?**
 
-    - It has advanced controls for creating loops, signaling, waiting for events, etc
-    - It's very simple to run scripts in a parallelized way
-    - It runs comands over SSH by design
+    - They are both workflow systems.
+    - They both run on your laptop and don't need a large server.
+    - You don't need to deploy them on target hosts or run them permanently.
+    - They can both run things on remote systems out of the box.
 
-    **What it does worse than Arcaflow**
+    **How is qDup different?**
 
-    - It does not have a typing system and cannot guarantee type-safety or validation of data
-    - It doesn't integrate with container engines natively
+    - qDup has advanced controls for creating loops, signaling, waiting for events, etc.
+    - qDup makes it very simple to run scripts in a parallelized way.
+    - qDup runs comands over SSH by design.
```

Comment:
Hi @dustinblack. qDup currently runs over SSH, locally (i.e on a local machine without the SSH requirement), or integrates with podman. There are plans to support docker/ocp and k8's in the future. https://github.com/Hyperfoil/qDup/blob/master/docs/reference/hosts.adoc